README.md

```diff
@@ -102,29 +102,27 @@ continuación se muestran ejemplos para una configuración de tipo standalone.
       enable = true;
       policies = {
         SecurityDevices = {
-          "OpenSC PKCS11" = "${pkgs.opensc}/lib/opensc-pkcs11.so";  # Para poder utilizar el DNIe, y otras tarjetas inteligentes
-          "DNIeRemote" = "${config.programs.dnieremote.finalPackage}/lib/libdnieremotepkcs11.so";  # Para poder utilizar el DNIe por NFC desde un móvil Android
+          "OpenSC PKCS11" = "${pkgs.opensc}/lib/opensc-pkcs11.so";  # To use DNIe and other smart cards
+          "DNIeRemote" = "${config.programs.dnieremote.finalPackage}/lib/libdnieremotepkcs11.so";  # To use DNIe via NFC from an Android mobile
         };
       };
       profiles.miperfil = {
-        id = 0;  # Hace que este perfil sea el perfil por defecto
-        # ... El resto de opciones de configuración de este perfil
+        id = 0;  # Makes this profile the default profile
```

Comment:
Caveat Emptor: If you have already created a default profile (perhaps non-declaratively), with implicit `id=0`, adding this will replace that profile, possibly making it acessible. Maybe suggesting to make a backup of that profile?

Co-authored-by: @CesarGallego

Reply:
Good point, but this may change with #22, so for now, we keep it like this.